BotRoyaleUnity/Assets/Scripts/PartHealth.cs

```diff
@@ -1,15 +1,20 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using System;
 
 public class PartHealth : MonoBehaviour
 {
-    public int health;
+    public float maxHealth = 20;
+
+    public Color initialColor;
+    public float health;
```

Comment:
initialColor and health should be private, since they shouldnt be configurable in the editor and other scripts shouldnt be reading or writing them directly